src/modules/shared/services/users.service.ts

```diff
@@ -96,36 +96,14 @@ export class UsersService extends CoreService {
     );
   }
 
-  searchUser(email: string): Observable<searchUserEndpointOutDTO[]> {
-    const roles =  [UserRoleEnum.ADMIN, UserRoleEnum.ACCESSOR, UserRoleEnum.ASSESSMENT, UserRoleEnum.INNOVATOR, UserRoleEnum.QUALIFYING_ACCESSOR]
+  /**
+   * Get's the information of a user through is email or id
+   * @param idOrEmail user id or email
+   */
+  getUserInfo(idOrEmail: string): Observable<UserInfo> {
```

Comment:
If this call is for the admin API, maybe it should be placed on a service of the admin module?

Only an admin can call this, right?

Reply:
Agree! Done in the latest commit!